models/models.js

```diff
@@ -1,5 +1,26 @@
 const db = require("../db/connection");
 
+/* ***************CHECK ARTICLE EXISTS****************/
+
+exports.checkArticleExists = async (articleID) => {
+  const value = [articleID];
```

Comment:
I'd argue that this could be passed straight in as [articleId] on line 14 to make things slightly DRYer.